lib/calendarClient.ts

```diff
@@ -506,13 +507,17 @@ const listCalendars = (withCredentials) =>
     results.reduce((acc, calendars) => acc.concat(calendars), [])
   );
 
-const createEvent = async (credential, calEvent: CalendarEvent): Promise<any> => {
+const createEvent = async (
+  credential,
+  calEvent: CalendarEvent,
+  emailTemplates: EmailTemplate[]
+): Promise<any> => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689304)
```suggestion
): Promise<unknown> => {
```